pandas/tests/frame/conftest.py

```diff
@@ -11,11 +11,16 @@
 import pandas._testing as tm
 
 
-@pytest.fixture(params=product([True, False], [True, False]))
+@pytest.fixture(params=["both", "neither", "left", "right"])
 def close_open_fixture(request):
     return request.param
 
 
+@pytest.fixture(params=product([True, False], [True, False]))
```

Comment:
dont' add a fixture for this, just test these directly

Reply:
will do :)